api/app/db/database.py

```diff
@@ -69,9 +95,11 @@ def get_read_session_scope() -> Generator[Session, None, None]:
         session.close()
 
 
-@ contextmanager
+@contextmanager
 def get_write_session_scope() -> Generator[Session, None, None]:
-    """Provide a transactional scope around a series of operations."""
+    """Provide a transactional scope around a series of operations.
+    THIS METHOD IS DEPRECATED! PLEASE MOVE TO USING: get_async_write_session_scope
```

Comment:
is it worthwhile creating a Task in ZenHub to remind ourselves to remove the sync session scope()s at some point? Or are you confident you/we will remember?

Reply:
It's going to be around a LOOOOONG time.... But sure - I'll create a dev task for migrating.